src/assets/sass/modules/WorkExperience/_work-experience.scss

```diff
@@ -0,0 +1,45 @@
+.work-experience-block{
+    margin-top: 25px;
+}
+
+.work-experience{
+    padding-top: 28px;
+    font: normal normal 14px/17px $font-primary;
+    color: mine-shaft;
+
+    @include e(row-header){
+        display: flex;
+        justify-content: space-between;
+        align-items: center;
+        padding-bottom: 4px;
+    }
+
+    @include e(exp-year){
+        color: $dusty-gray; 
+        padding: 6px 0 12px;
+    }
+
+    @include e(row-item){
+        padding: 0 0 20px 12px;
+    }
+
+    @include e(row){
+        &:last-child{
+            padding-bottom: 28px;
+            border-bottom: 1px solid $alto;
+        }
+    }
+
+    @include m(hidden){
+        color: $dusty-gray;
+    }
+
+    .hidden-tag{
```

Comment:
This is used only in one place, and as a BLOCK. If this is meant to be used as a BLOCK then you can add this in file `_global.scss` or make a separate partial as `_hidden-tag.scss`.
Since this is a tag, you can rename this class to `tag tag--hidden` instead of `hidden-tag`, but this depends on the purpose of the tag.

Reply:
🆗